cypress/e2e/08-account_creation.cy.js

```diff
@@ -53,15 +50,13 @@ Cypress.Commands.add("enterPhoneNumber", () => {
 
 Cypress.Commands.add("scanQrCode", () => {
   cy.contains("Get your security code via");
-  cy.injectSRAxe();
   cy.checkA11y();
   cy.get(submitButton).click();
 });
 
 Cypress.Commands.add("verifySecurityCode", (code) => {
   cy.contains("Verify your security code.");
   cy.contains('One-time security code');
-  cy.injectSRAxe();
   cy.checkA11y();
```

Comment:
What's the rule for where you've removed or added the calls to `injectSRAxe`? Is it redundant with the `checkA11y` call right below?

Reply:
you need to inject the `axe` runtime before you run `cy.checkA11y()` and typically its recommended to inject after you visit the page. it was injecting it multiple times after `cy.injectSRAxe()` multiple times after you visit a page, now we only inject it once.

Reply:
I didn't know this either.